src/executor_v2/explain.rs

```diff
@@ -14,12 +15,17 @@ pub struct ExplainExecutor {
 
 impl ExplainExecutor {
     pub fn execute(self) -> BoxedExecutor {
-        let explain = format!(
-            "{}",
-            Explain::of(&self.plan)
-                .with_costs(&costs(&self.plan))
-                .with_catalog(&self.catalog)
-        );
+        let binding = costs(&self.plan);
+        let binding = Explain::of(&self.plan)
```

Comment:
Why called 'binding'?

Reply:
I think it's not intended. I'll rename them. They're generated by rust analyzer